### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by prasanthpul**
This includes changes from #3097 which can be discarded if this is merged.

### CODE REVIEW

**Review by askhade — APPROVED** [review]

**Review by gramalingam — APPROVED** [review]

**Review by jcwchen — APPROVED** [review]
LGTM. Thank you.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+If you have old commits that are not signed, use the following commands to squash the old PR (original branch) into a single commit. This is an easier way to signoff old commits in old PR.
+	
+```bash
+git checkout master
+git checkout -b temporary_patch              # create a new branch as temporary
+git merge --squash original_patch            # copy from old branch
+git branch -d original_patch                 # remove old branch
+git checkout -b original_patch               # create a new branch with the same name (override)
+git commit -m ‘type your own commit msg’ -s  # signoff that single commit
```
Comment (jcwchen) [inline CONTRIBUTING.md:44]:
```suggestion
git commit -m 'type your own commit msg' -s  # signoff that single commit
```

Comment (jcwchen) [reply CONTRIBUTING.md:44]: Wrong character

Comment (prasanthpul) [reply CONTRIBUTING.md]: updated

**Pull request merged**